File: demo1s/src/main/scala/org/mosquito/demo1s/query/UserQuery.scala
package org.mosquito.demo1s.query

import org.mosquito.framework.core.model.BaseQuery

import scala.beans.BeanProperty


/**
 * 分页查询参数实体
 *
 * @author jcchen
 * @Date 2018-5-28
 * 
 */
class UserQuery extends BaseQuery {


    /**
     * 
     * 
     */
    @BeanProperty var name: String = _

    /**
     * 
     * 
     */
    @BeanProperty var password: String = _

    /**
     * 
     * 
     */
    @BeanProperty var salt: String = _

    /**
     * 
     * 
     */
    @BeanProperty var mobile: String = _
	
}

File: demo4/src/main/java/org/mosquito/demo4/query/UserQuery.java
package org.mosquito.demo4.query;

import org.mosquito.framework.core.model.BaseQuery;

import lombok.Getter;
import lombok.Setter;


/**
 * 分页查询参数实体
 *
 * @author jcchen
 * @Date 2018-6-6
 * 
 */
@Getter
@Setter
public class UserQuery extends BaseQuery {

    private static final long serialVersionUID = 1L;

    /**
     * 
     * 
     */
    private String name;

    /**
     * 
     * 
     */
    private String password;

    /**
     * 
     * 
     */
    private String salt;

    /**
     * 
     * 
     */
    private String mobile;
	
}
